Dockerfile.ci

```diff
@@ -1068,12 +1068,6 @@ function check_run_tests() {
         python "${IN_CONTAINER_DIR}/remove_arm_packages.py"
     fi
 
-    if [[ ${TEST_TYPE} == "PlainAsserts" ]]; then
-       # Plain asserts should be converted to env variable to make sure they are taken into account
-       # otherwise they will not be effective during test collection when plain assert is breaking collection
-       export PYTEST_PLAIN_ASSERTS="true"
-    fi
```

Comment:
Nice to not need this special case anymore!

Reply:
Yes. It mysteriously stopped happening :)